faculty.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'human'
+
+class Faculty < Human
+  attr_reader :age, :salary, :phase, :target_raise
+  attr_accessor :name
+
+  def initialize (options = {})
+
+  end
```

Comment:
If you declare an empty `initialize`, nothing will happen when you call `.new`!  Either leave it out and let the inheritance lookup chain do its thing, or explicitly call `super`.